5_rest_requests_single_model.ipynb

```diff
@@ -1,13 +1,15 @@
 {
  "cells": [
   {
-   "cell_type": "code",
+   "cell_type": "markdown",
    "execution_count": null,
    "id": "55c8afde-9b18-4b6a-9ee5-33924bdb4f16",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# REST Inference"
+    "# REST Inference for single-model server deployment \n",
+    "\n",
+    "_Note: Use this procedure for testing a model that you deployed on a multi-model server. See `3_rest_requests_multi_model.ipynb` for testing a model that you deployed on a multi-model server._"
```

Comment:
Should this be: "_Note: Use this procedure for testing a model that you deployed on a single-model server" ?

Reply:
yes